### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by joshski** [bbc/bbc-a11y#158]
bbc-a11y has only been tested on OSX and Linux. Windows should be easy to support, but ideally we need a continuous build to keep it working.

**Comment by joshski** [comment]
I remember there were some issues regarding BBC using particular CI services. Does anybody know if we can use AppVeyor?

https://www.appveyor.com/

They offer a windows-based continuous integration service that's free for open source projects.

**Comment by EmmaJP** [comment]
Let's ask on Slack.

**Comment by joshski** [comment]
I did. Tumbleweed. Maybe I'm #general 😄

**Comment by EmmaJP** [comment]
@andymsuk @joshski do you know if the suggested move to Chrome headless #282 would also provide better support for windows users?

**Comment by andymsuk** [comment]
@EmmaJP I'd imagine so, yes.
